scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -801,7 +801,7 @@ def add_parse_arguments(parser = None) -> argparse.ArgumentParser:
         "--log-level",
         type=str.upper,
         default='INFO',
-        choices=['NOTSET', 'DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
+        choices=['NOTSET', 'DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL', 'STEP', 'QA'],
```

Comment:
```suggestion
        choices=['NOTSET', 'DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL', 'TRACE', 'SANITY'],
```
imo these are better reflect the purpose.